build.gradle

```diff
@@ -220,6 +215,57 @@ jar {
     }
 }
 
+macAppBundle {
+    mainClassName	= "pcgen.system.Main"
+    appName			= "PCGen"
+    appStyle		= "universalJavaApplicationStub"
+    icon 			= "installers/mac-installer/PCGenApp.icns"
+    bundleJRE 		= false
```

Comment:
IMHO we should bundle it: many users only have Java 8.

Reply:
Neither Java 10 nor Java 11 have a JRE; that is not more.  What needs to happen with Java 11 is a custom runtime be built using the new modular approach.  The DMG plugin does not support that, and neither does PCGen. The DMG plugin doesn't verify that it's a JRE, so, a custom Mac runtime can be created and bundled.

In the meantime, if a user does not have Java 11, they will git a dialog first asking to allow system notifications, and then informing them that Java 11 needs to be installed.

Reply:
Actually look deeper, we are bundling a Java 11 JRE from AdoptOpenJDK.